apps/charterafrica/src/lib/payload/index.js

```diff
@@ -21,17 +21,37 @@ async function getCollection(collection, options) {
   });
 }
 
+async function createCollection(collection, data, options) {
+  return payload.create({
+    collection,
+    data,
+    ...options,
+  });
+}
+
 async function findGlobal(slug, options) {
   return payload.findGlobal({
     ...options,
     slug,
   });
 }
 
+async function updateCollection(collection, id, data, options) {
+  const { id: _, ...rest } = data;
+  return payload.update({
+    collection,
+    id,
+    data: rest,
+    ...options,
+  });
```

Comment:
```suggestion
  return payload.update({
    ...options,
    collection,
    id,
    data,
  });
```

1. We've already made id a standalone arg,
2. `options` shouldn't override `collection`, `id`, etc.